pkg/chain/ethereum/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/hashcloak/Meson-plugin/pkg/chain/ethereum
+
+go 1.13
```

Comment:
Is a new go module really neccesary?

It can already be imported in other packages with:
```go
import (
      meson-eth "github.com/hashcloak/Meson-plugin/pkg/chain/ethereum"
)
```

If you do think it might be needed I think we should wait until that time comes just to avoid having to avoid a bunch of repos if we decide to change where this module is located.

Reply:
I'm not a gopher, so I'm still wrangling with go package management. I thought to import the way you just did in your example, I needed to `go mod init`. You're saying that I don't need to do that? I know for sure that if I'm using the package locally, I don't but I wasn't too sure for external packages.

Reply:
Yeah, you don't need the init that module. A quick example of how it works can be seen here: https://github.com/sebohe/basic-cosmos-signing/blob/master/main.go#L9 and https://github.com/binance-chain/go-sdk/tree/master/client/rpc

What matter more is `package <name>` at the top.

Reply:
Thanks